doc/api/tls.md

```diff
@@ -16,7 +16,8 @@ const tls = require('tls');
 
 ## TLS/SSL concepts
 
-The TLS/SSL is a public/private key infrastructure (PKI). For most common
+TLS/SSL is a set of protocols that rely on a public key infrastructure (PKI) to
```

Comment:
Not necessary to fix it in this PR, but in case you're feeling like you want to be thorough: PKI appears twice in this document. This is the first instance, so spelling it out makes sense. Later, it appears as `PKI (public key infrastructure)` but just "PKI" would be sufficient, since we define it here. Or at least let's change it to the format you have here (`public key infrastructure (PKI)`) for consistency (and conformance with the Microsoft style guide).